test/instances/aave/AaveHyperdrive.t.sol

```diff
@@ -110,12 +110,12 @@ contract AaveHyperdriveTest is InstanceTest {
     /// @param _factory The address of the Hyperdrive factory.
     function deployCoordinator(
         address _factory
-    ) internal override returns (address) {
+    ) public override returns (address) {
         vm.startPrank(alice);
```

Comment:
I seem to be having trouble calling this contract's functions. My best guess is that I'm not using the `alice` account correctly from this function. Any chance we can either (1) expose the private key of `alice`, or (2) allow the account as an argument?